telemetry/init.go

```diff
@@ -0,0 +1,42 @@
+package telemetry
```

Comment:
Is there any specific reason why this is top-level in the repo, and not a tm2 package?

Reply:
Yeah, so metrics can be collected from any package in this repo